scripts/speech_recognition/convert_to_tarred_audio_dataset.py

```diff
@@ -174,6 +174,11 @@
         "and it must be filled out by the user."
     ),
 )
+parser.add_argument(
+    "--shard_manifests",
+    action='store_false',
```

Comment:
You don't need --shard_manifests = true like in the example usage. just --shard_manifest is enough

Reply:
fixed the PR description, thanks!